build/opentsdb/start-opentsdb.sh

```diff
@@ -6,8 +6,14 @@ export COMPRESSION=NONE
 export HBASE_HOME=/opt/hbase
 export JAVA_HOME=/usr/lib/jvm/jre
 
-# Overwrite the default open tsdb config if available
 true ${OPENTSDB_RESOURCE:=/usr/local/serviced/resources/opentsdb}
+
+# Overwrite the default metric consumer config if available
+if [ -e "$OPENTSDB_RESOURCE/metric-consumer-app/configuration.yaml" ]; then
```

Comment:
I need to release the new isvcs before I can update serviced.  How do you want me to change the config file?